src/lib/loaders/loaders_without_authentication/vortex.ts

```diff
@@ -0,0 +1,50 @@
+import { executableVortexSchema } from "lib/stitching/vortex/schema"
+import gql from "lib/gql"
+import { graphql } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { error } from "util"
+
+const schema = executableVortexSchema({ removePricingContext: false })
+const query = gql`
+  query artworkPricingContextQuery(
+    $artistId: String!
+    $category: AnalyticsPricingContextCategoryEnum!
+    $widthCm: Int!
+    $heightCm: Int!
+  ) {
+    analyticsPricingContext(
+      artistId: $artistId
+      category: $category
+      widthCm: $widthCm
+      heightCm: $heightCm
+    ) {
+      bins {
+        maxPriceCents
+        minPriceCents
+        numArtworks
+      }
+      filterDescription
+    }
+  }
+`
+
+export default () => {
+  return {
+    async pricingContextLoader(
+      vars: {
+        artistId: string
+        category: string
+        widthCm: number
+        heightCm: number
+      },
+      context: ResolverContext
+    ) {
+      const result = await graphql(schema, query, null, context, vars)
```

Comment:
To all those on this PR; when in the future you see the usage of the `graphql` function, it is perhaps incorrectly trying to implement stitching. What is being done here is more a gateway to an upstream service and doesn’t lead to stitching advantages, i.e. we’re selecting _all_ of the fields, rather than only the ones being requested by the client. We don’t want to do this going forward 🙏 

Probably the _only_ case where we have to use this function as a gateway is when we have a _field_ in MP whose value is derived from one or more upstream values (and even there we haven’t spend a lot of time investigating the options within the stitching framework).